sources/models/src/modeled_types/kubernetes.rs

```diff
@@ -1002,6 +1006,177 @@ mod test_topology_manager_policy {
 
 // =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=
 
+/// imageGCHighThresholdPercent is the percent of disk usage after which image
+/// garbage collection is always run. The percent is calculated by dividing this
+/// field value by 100, so this field must be between 0 and 100, inclusive. When
+/// specified, the value must be greater than imageGCLowThresholdPercent.
+/// Default: 85
+/// https://kubernetes.io/docs/reference/config-api/kubelet-config.v1beta1/
+
+#[derive(Debug, Clone, Eq, PartialEq, Hash)]
+pub struct ImageGCHighThresholdPercent {
+    // All settings are brought in as Strings.
+    // Important note: custom types need to have `string_impls_for!()` after
+    // their TryFrom<&str> implementation. (this requires that the inner type
+    // of a custom Kubernetes settings type is a String)
+    inner: String,
+}
+
+impl TryFrom<&str> for ImageGCHighThresholdPercent {
+    type Error = error::Error;
+
+    fn try_from(input: &str) -> Result<Self, Self::Error> {
+        // apiserver validation 1: reject empty values
+        ensure!(
+            !input.is_empty(),
+            error::InvalidImageGCHighThresholdPercentSnafu {
+                input,
+                msg: "must not be empty",
+            }
+        );
+        // apiserver validation 2: reject values that are out of bounds
```

Comment:
Both of these comments seem like the "restate what the code is doing" variety. Usually these aren't necessary.

```suggestion
        ensure!(
            !input.is_empty(),
            error::InvalidImageGCHighThresholdPercentSnafu {
                input,
                msg: "must not be empty",
            }
        );
```